### DISCUSSION TITLE
Will create process activate (App Lifecycle) allow Steam distribution?

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [microsoft/WindowsAppSDK#1048] [Q&A]
The top issue with UWP apps (or perhaps second top, after .Net support) is their inability to be distributed via Steam.

Does this capability make that possible? If not, what does? And is it on the roadmap?

**Comment by maxkatz6** [comment]
Probably I don't know something, but what is the problem with steam? Steam should support at least anything from activation protocols that UWP provide. 
This SDK adds unified ways to activate application, especially installed with MSIX, but once again - Steam needs to support it. But so far it seems they are extremely ignorant.

**Reply by Gavin-Williams** [reply]
Has anyone tried to put a UWP game onto steam? Or does the games industry see UWP as a completely dead end technology?

I have played some UWP games from the MS Store, but none on steam.

**Comment by Gavin-Williams** [comment]
The whole point of packaged apps is security and trust. Reducing the risk of malware on our own computers, and also reducing malware in the community. If game development is ignored on Windows, then malware continues to be rife in that market, which kind of defeats the purpose. Sure, some protection is good, but we should know by now, that _some_ just isn't good enough.

**Reply by ghost** [reply]
> The whole point of packaged apps is security and trust. Reducing the risk of malware on our own computers, and also reducing malware in the community.

misconception. where did you derive that from ?
 
* **the whole point of **_packaged apps_** is `Clean Install and Uninstall`.** (and previously APIs that required identity, now this requirement is lifted). **Nothing more , nothing less** .

* **The security and trust you are talking about is provided by NT kernel's** Low IL's another variant known as **`AppContainer`. Nothing more, nothing less**.    

You can run any ancient DOS/Win32/UWP/Packaged/Unpackaged/.NET/WSL 1's  POSIX  apps through AppContainer and it will immediately be subjected to under **that same Security and Trust.**
